scrapy/downloadermiddlewares/retry.py

```diff
@@ -70,8 +72,10 @@ def _retry(self, request, reason, spider):
             retryreq.meta['retry_times'] = retries
             retryreq.dont_filter = True
             retryreq.priority = request.priority + self.priority_adjust
+            self.stats.inc_value('retry/count')
```

Comment:
I can see though how it makes it more obvious for the user what caused these retries